Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -157,15 +157,16 @@ public NetworkManager(IConfigurationManager configurationManager, ILogger<Networ
         /// Creates a new network collection.
         /// </summary>
         /// <param name="source">Items to assign the collection, or null.</param>
+        /// <param name="unique"><c>True</c> if subnets that overlap should be merged (default).</param>
         /// <returns>The collection created.</returns>
-        public static Collection<IPObject> CreateCollection(IEnumerable<IPObject>? source = null)
+        public static Collection<IPObject> CreateCollection(IEnumerable<IPObject>? source = null, bool unique = true)
```

Comment:
There's 5 usages of `CreateCollection` in NetworkManager and 1 in DlnaEntryPoint. All 5 in NetworkManager are called with false. Is it the correct default then?

Reply:
Thought there were more.

Have removed the extra param from CreateCollection, and hardcoded value in addItem.